source/leuven/genius_2_rocky.rst

```diff
@@ -1,121 +1,267 @@
-~~~~~~~~~~~~~~~~~~~~~
+e.g.:
 
-Instead of working with a single default, unchanging, modulepath, refering to a single default toolchain, a new approach is taken. Each cluster partition will have a so called cluster module that sets the ``$MODULEPATH`` that is valid for the specific nodes in the cluster partition. The cluster module will detect the underlying CPU architecture and uses this for setting the path correctly.
+.. code-block:: shell
 
+   /apps/leuven/rocky8/skylake/2018a/modules/all
 
-.. _check_available_software:
+This convention is in line with other VSC sites and will also be used on wICE
+and future clusters. In order to add such a directory to your modulepath, the
+following command can be used:
 
-Check available software
-~~~~~~~~~~~~~~~~~~~~~~~~
-On the login node you will be able to load the different available cluster modules (only for experimentation, not for actual computing). In order to do this you can query the available cluster moduels with::
+.. code-block:: shell
 
-   $ module avail
+   module use /apps/leuven/rocky8/skylake/2018a/modules/all
 
-this wil show the available cluster modules::
+To remove the entry again:
 
-   --------------------------- /apps/leuven/etc/modules ---------------------------
-   cluster/default
-   cluster/genius/amd_long               (S)
-   cluster/genius/amd                    (S)
-   cluster/genius/batch_debug            (S)
-   cluster/genius/batch_long             (S)
-   cluster/genius/batch                  (S)
-   ...
-   cluster/wice/batch
-   ...
+.. code-block:: shell
 
-Loading any of this modules on the login node::
+   module unuse /apps/leuven/rocky8/skylake/2018a/modules/all
 
-   $ module load cluster/genius/batch
+.. _impact_on_user_installed_software:
 
-will set the module path for the modules that are applicable for the Genius ``batch`` partition::
+Impact on user-installed software
+---------------------------------
+If you have installed a software package yourself in your own account, and you
+did this on a Genius CentOS 7 node, it must be recompiled on Genius on a node
+with the new OS. This can be done on one of the available test nodes. Please
+request access to the lpt2_rocky8_pilot group.
 
-   $ module load cluster/wice/batch
+Conda environments
+----------------------------
+The Conda environment you installed might need reinstallations. If you already
+have a Conda environment that works on wICE, it also should work on Genius
+after the migration. If you only have a Conda environment working on Genius,
+it is best to create a new Conda installation. In this case, it is recommended
+to recreate your environment for full compatibility with the new OS. Best practice
+is to choose a new installation folder with explicit mention of the new OS, e.g.::
 
-will set the the module path of the wICE ``batch`` partition. When you do this on the login node you can examine wich modules are available with the regular commands, e.g.::
+   ${VSC_DATA}/miniconda3-rocky
```

Comment:
Given that Rocky9 is already out and we are sticking to Rocky8, why not adding the OS version to the path, too? It is just an extra character, but can accommodate possible OS upgrades in the future, too.

Reply:
That would be indeed a  more futureproof choice